css/app.css

```diff
@@ -35,3 +35,28 @@ gaia-list .destructive {
 a.navigation *, a.action *, a.edit * {
   pointer-events: none;
 }
+
+gaia-list span.group {
```

Comment:
ignore my comment, I don't really care. But let's keep this in mind if we want it to be reused in the future.